visavis_vision/ObstacleInfo[] obstacles